portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +27,29 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Tino!</h1>");
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("Hi tino");
+    comments.add("i like pie");
+    comments.add("nice space");
```

Comment:
It's best practice to define constants like these as static variables at the top.

Reply:
Sounds good! Removed constants, and I'm now using GSON! Thank you!